deepchem/models/wandblogger.py

```diff
@@ -17,9 +17,28 @@ class WandbLogger(object):
     will log the specified metrics calculated on the specific datasets
     to the user's W&B dashboard.
 
-    If a WandbLogger is provided to the wandb_logger flag,
+    If a WandbLogger is provided to the wandb_logger flag of the model,
     the metrics are logged to Weights & Biases, along with other information
     such as epoch number, losses, sample counts, and model configuration data.
+
+    Usage
```

Comment:
Typically we use 
```
Examples
--------
```
for the usage examples (following the numpy docstyle)

Reply:
Fixed!